### PR TITLE
Update experiment API

### PR DESCRIPTION
**Pull request by quanpython**

### CODE REVIEW

File: studio/alembic/versions/8f5d1e7d2a25_add_experiment_view_attributes.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""add experiment view attributes
```
Comment (itutu-tienday) [inline studio/alembic/versions/8f5d1e7d2a25_add_experiment_view_attributes.py:1]: @quanpython 
I have committed a file similar to this one on the latest develop-main branch, so there is no problem deleting it.
(Please merge develop-main)

File: studio/app/optinist/routers/expdb.py [hunk 2]
```diff
@@ -120,6 +128,45 @@ def get_cell_urls(source, exp_dir, index: int, params=None):
     ]
 
 
+def get_search_db_experiment_query(
```
Comment (itutu-tienday) [inline studio/app/optinist/routers/expdb.py:131]: In the data registration process of view_attributes, I assume that multibyte characters are escaped (ex: "imaging_depth": "220µm" -> "280\u00b5m"), so I assume that the search string needs to be escaped.
Is escaping at the frontend or the backend the preferred form? (Consultation)

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: studio/app/optinist/routers/expdb.py (hunk 2 above)
Comment (quanpython) [reply studio/app/optinist/routers/expdb.py:131]: @itutu-tienday I think it should be handled at the frontend (same as #246)

**Comment by itutu-tienday** [comment]
@quanpython 
Please update this PR with open if status update OK.

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**